workflows/RNA-seq/RNA-seq

```diff
@@ -283,7 +290,22 @@ def main():
     ## save to configs.yaml in outdir
     cf.write_configfile(os.path.join(args.outdir,'config.yaml'),config)
 
-    snakemake_module_load = "module load snakemake/3.12.0 slurm &&".split()
+    ## cluster config
+    cluster_config = cf.load_configfile(os.path.join(this_script_dir,"../../shared/","cluster.yaml"),False)
+    cluster_config = cf.merge_dicts(cf.load_configfile(os.path.join(this_script_dir,"cluster.yaml"),False), cluster_config)
+    
+    if args.cluster_configfile:
+        user_cluster_config = cf.load_configfile(args.cluster_configfile,False)
+        cluster_config = cf.merge_dicts(cluster_config, user_cluster_config) # 2) form user_config.yaml
+    cf.write_configfile(os.path.join(args.outdir,'cluster_config.yaml'),cluster_config)
+    
+    snakemake_module_load = "module load snakemake/3.12.0 "
+    
+    if not args.local:
+        snakemake_module_load += cluster_config["cluster_module_load"]
+    
+    snakemake_module_load = (snakemake_module_load + " && ").split()
+    
     snakemake_cmd = """
                     snakemake {snakemake_options} --latency-wait 300 --snakefile {snakefile} --jobs {max_jobs} --directory {outdir} --configfile {configfile}
```

Comment:
Might want to make `--latency-wait` configurable. Some file systems might need a longer value.

Reply:
yes, would be good as well